lib/messager.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+require 'date'
+
+class Messager
+
+  def send_text
+    account_sid = "ACc640aebc9ed667fe38f3481bab876368"
+    auth_token = "134e02cb4127679d53aaed26bd40af7d"
+    client = Twilio::REST::Client.new account_sid, auth_token
+
+    from = "+441702806346" # Your Twilio number
```

Comment:
use dotenv gem to hide personal details on github https://github.com/makersacademy/takeaway-challenge/blob/master/docs/review.md#personal-details-and-tokens-on-github  these details should be put in a .env file which should be added to your .gitignore so it is not pushed to github